### DISCUSSION TITLE
Backward support?

### DISCUSSION THREAD

**Discussion by bobyang3** [sec/dotnet-core-freebsd-source-build#33] [Q&A]
If I install version 8.xxx, will I be able to run .net 6 app?
Thanks

**Comment by sec** [comment]
Hi,
It could be possible in theory, but it might not work as expected. That's a more general question regarding .net itself, not the builds I make - you could check https://github.com/dotnet/runtime/issues/79650 and https://learn.microsoft.com/en-us/dotnet/core/compatibility/categories - as 6 vs 8 is major version, thing could break.
